src/components/menu/menu.tsx

```diff
@@ -65,6 +65,16 @@ export default function Menu() {
     closeMenu();
   }, []);
 
+  const playLogo = useCallback(() => {
+    const logo = document.getElementById("logo") as HTMLImageElement | null;
+    if (logo) {
```

Comment:
`if (logo == null)` (en javascript truthiness and falsiness sont un peu ambigus, c'est mieux d'etre explicite)

Reply:
logo != null ?